application/search/filters.py

```diff
@@ -38,6 +38,10 @@ class DatasetQueryFilters:
 @dataclass
 class QueryFilters:
     # base filters
+    exclude_field: Optional[List[str]] = Query(
```

Comment:
Maybe move next to the field parameter as they're closely related. Could be. worth adding something like:

field parameter will over ride any fields given in the exclude_field parameter. So that the user knows the expected behaviour when. both are given